maxim/logger/llamaindex/__init__.py

```
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional, Sequence
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)